### PR TITLE
Carousel UI

### PR DESCRIPTION
**Pull request by santhoshz-developer**

### CODE REVIEW

File: website/package.json [hunk 1]
```diff
@@ -12,12 +12,15 @@
     "next": "14.2.3",
     "react": "^18",
     "react-dom": "^18",
+    "react-slick": "^0.30.2",
```
Comment (vijayaprathapkumar) [inline website/package.json:15]: why  have you used react-slick ?

File: website/src/components/customerReview/CustomerReview.Styled.tsx [hunk 2]
```diff
@@ -0,0 +1,139 @@
+import breakpoints from "@/themes/breakpoints";
+import { theme } from "@/themes/theme";
+import { styled } from "styled-components";
+
+export const TestimonialCard = styled.div`
+  background: ${theme.colors.white};
+  padding: 2rem;
+  border: 1px solid #e4e4e4;
```
Comment (vijayaprathapkumar) [inline website/src/components/customerReview/CustomerReview.Styled.tsx]: use color form theme

File: website/src/components/customerReview/CustomerReview.Styled.tsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  height: 280px;
```
Comment (vijayaprathapkumar) [inline website/src/components/customerReview/CustomerReview.Styled.tsx]: Do not mention height

File: website/src/components/customerReview/CustomerReview.Styled.tsx [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+export const AuthorName = styled.h3`
+  font-size: 1.2rem;
+  font-weight: bold;
+  color: ${theme.colors.grey};
+`;
+
+export const AuthorTitle = styled.p`
+  font-size: 0.9rem;
+  color: #666;
```
Comment (vijayaprathapkumar) [inline website/src/components/customerReview/CustomerReview.Styled.tsx]: use colot from theme

File: website/src/components/customerReview/CustomerReview.Styled.tsx [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+  border-radius: 50%;
+  padding: 20px;
+  background-color: ${theme.colors.white};
+  background-image: url("images/arrowLeft.png");
+  cursor: pointer;
+  color: ${theme.colors.white};
+  background-repeat: no-repeat, repeat;
+  &:hover {
+    background-color:  ${theme.colors.white};
+    background-image: url("images/arrowLeft.png");
```
Comment (vijayaprathapkumar) [inline website/src/components/customerReview/CustomerReview.Styled.tsx:64]: why are you using same background image on hover

File: website/src/components/customerReview/CustomerReview.Styled.tsx (hunk 2 above)
Comment (santhoshz-developer) [reply website/src/components/customerReview/CustomerReview.Styled.tsx]: fix it sir

File: website/src/components/customerReview/CustomerReview.Styled.tsx (hunk 3 above)
Comment (santhoshz-developer) [reply website/src/components/customerReview/CustomerReview.Styled.tsx]: resolve it sir

File: website/src/components/customerReview/CustomerReview.Styled.tsx (hunk 4 above)
Comment (santhoshz-developer) [reply website/src/components/customerReview/CustomerReview.Styled.tsx]: already resolve another branch sir

File: website/src/components/customerReview/CustomerReview.tsx [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+  TestimonialAuthor,
+  TestimonialCard,
+  TestimonialText,
+  Title,
+  Warrper,
+} from "./CustomerReview.Styled";
+import { CustomItems } from "@/config/carousal";
+
+const CustomerReviewComponent = () => {
+  const Testimonial = ({ text, author, title, imageSrc }: any) => (
```
Comment (vijayaprathapkumar) [inline website/src/components/customerReview/CustomerReview.tsx]: Do not use any

File: website/src/components/enviormentPage/EnvironmentPage.Styled.tsx [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 20px;
+  color: black;
```
Comment (vijayaprathapkumar) [inline website/src/components/enviormentPage/EnvironmentPage.Styled.tsx]: Get colors from theme

File: website/src/components/enviormentPage/EnvironmentPage.Styled.tsx [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+  width: 32px;
+  height: 32px;
+`;
+export const Title = styled.h1`
+  color: ${theme.colors.white};
+`;
+
+export const SliderContainer = styled.div`
+  margin: 0 auto;
+  width: 800px;
```
Comment (vijayaprathapkumar) [inline website/src/components/enviormentPage/EnvironmentPage.Styled.tsx]: No nedd to provide with 800px

File: website/src/components/service/ServicePage.styled.tsx [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+  color: ${theme.colors.white};
+  font-size: 20px;
+  margin-bottom: 10px;
+`;
+
+export const ServiceTitle = styled.h3<{ textColor?: string }>`
+  font-size: 18px;
+  font-weight: bold;
+  margin-bottom: 5px;
+  color: ${(props) => props.textColor || "black"};
```
Comment (vijayaprathapkumar) [inline website/src/components/service/ServicePage.styled.tsx]: use black from theme

File: website/src/components/service/ServicePage.styled.tsx [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+`;
+
+export const ServiceIcon = styled.div<{ bgColor?: string }>`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  background-color: ${(props) => props.bgColor || "white"};
```
Comment (vijayaprathapkumar) [inline website/src/components/service/ServicePage.styled.tsx]: use white from theme

File: website/src/components/service/ServicePage.styled.tsx [hunk 11]
```diff
@@ -0,0 +1,147 @@
...
+  }
+`;
+export const Header = styled.div`
+  font-size: 18px;
+  font-weight: 600;
+  color: ${theme.colors.green};
+`;
+
+export const ServiceCard = styled.div<{ cardBgColor?: string }>`
+  background-color: ${(props) => props.cardBgColor || "white"};
```
Comment (vijayaprathapkumar) [inline website/src/components/service/ServicePage.styled.tsx]: use black from theme

File: website/src/components/service/ServicePage.styled.tsx [hunk 12]
```diff
@@ -0,0 +1,147 @@
...
+  font-weight: bold;
+  margin-bottom: 5px;
+  color: ${(props) => props.textColor || "black"};
+`;
+
+export const ServiceDescription = styled.p<{ textColor?: string }>`
+  font-size: 14px;
+  margin-top: 20px;
+  line-height: 1.5;
+  color: ${(props) => props.textColor || "black"};
```
Comment (vijayaprathapkumar) [inline website/src/components/service/ServicePage.styled.tsx]: use black from theme

File: website/src/components/customerReview/CustomerReview.tsx (hunk 6 above)
Comment (santhoshz-developer) [reply website/src/components/customerReview/CustomerReview.tsx]: fix it sir

File: website/src/components/enviormentPage/EnvironmentPage.Styled.tsx (hunk 8 above)
Comment (santhoshz-developer) [reply website/src/components/enviormentPage/EnvironmentPage.Styled.tsx]: fix it sir

File: website/src/components/service/ServicePage.styled.tsx (hunk 9 above)
Comment (santhoshz-developer) [reply website/src/components/service/ServicePage.styled.tsx]: fix it sir

**Pull request merged**